### PR TITLE
WFLY-6578: updated XSD

### PR DESCRIPTION
**Pull request by matlach**

### CODE REVIEW

**Comment by bstansberry** [comment]
this is ok to test

File: connector/src/main/resources/schema/wildfly-datasources_4_0.xsd [hunk 1]
```diff
@@ -1033,8 +1033,9 @@
       <xs:annotation>
         <xs:documentation>
           <![CDATA[[
-            Specifies the name of AS7 module providing this driver.
-            Thios tag is not used in IronJacamar standalone container.
+            Specifies the name of JBoss module providing this driver.
```
Comment (bstansberry) [inline connector/src/main/resources/schema/wildfly-datasources_4_0.xsd]: Nit: I think "JBoss Modules module" or just "module" is better.

Comment (matlach) [inline connector/src/main/resources/schema/wildfly-datasources_4_0.xsd]: should I do another pull request reflecting this change?

Comment (bstansberry) [inline connector/src/main/resources/schema/wildfly-datasources_4_0.xsd]: Sorry, I didn't see this question. It's fine to just update this branch and push it again to your github repo. The pull request will automatically be updated. Best is to add another commit and then user "git rebase -i" and squash the commits into 1.

**Comment by kabir** [comment]
@matlach As this pull request has  been held up for a long time, I am closing it to get the pull request queue under control again. Since this is very old now, it will be a very long time until this one will be addressed. Also, I don't have much time to spend on this to evaluate on a case by case basis.

If this is something important to you or the project please feel free to recreate the pull request. That way it will come on top of the pile again, and be looked at again sooner. 

If it is a new **feature**, please follow the new process for features. Ping @kabir or @jamez if you are unsure what this means, and we can help. Please do that before opening the pull request though.

Sorry for any inconvenience!
